scipy/stats/_binned_statistic.py

```diff
@@ -356,7 +356,8 @@ def binned_statistic_2d(x, y, values, statistic='mean',
 
 
 def binned_statistic_dd(sample, values, statistic='mean',
-                        bins=10, range=None, expand_binnumbers=False):
+                        bins=10, range=None, binnumbers=None,
+                        expand_binnumbers=False):
```

Comment:
same comment here--`binnumbers` arg may need to be moved to the end if experts find this approach helpful

Reply:
This has been solved.